malcolm/modules/excalibur/blocks/excalibur_detector_runnable_block.yaml

```diff
@@ -85,3 +85,12 @@
 
 - ADCore.parts.DatasetTablePart:
     name: DSET
+
+- excalibur.blocks.excalibur_munge_block:
```

Comment:
Why is this needed? Is this just so you can disable the VDS writer and it still works?